R/database.R

```diff
@@ -1520,29 +1599,28 @@ get_ssm_by_region = function(chromosome,
         maf_partial_path = config::get("results_flatfiles")$ssm$template$merged$augmented
       }else{
         maf_partial_path = config::get("results_flatfiles")$ssm$template$merged$deblacklisted
-        }
+      }
     }else if (mode == "strelka2"){
       maf_partial_path = config::get("results_flatfiles")$ssm$all$strelka2
```

Comment:
In the section above we need to get rid of the hard-coded path. Can you put this path in a new section in the config file?

tabix_bin = "/home/rmorin/miniconda3/bin/tabix"

e.g. 
default:
    dependencies:
        tabix: "/home/rmorin/miniconda3/bin/tabix"

Reply:
Will do!